msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java

```diff
@@ -40,13 +40,15 @@
 import com.microsoft.identity.labapi.utilities.constants.Mfa;
 import com.microsoft.identity.labapi.utilities.constants.TempUserType;
 
+import org.junit.Ignore;
 import org.junit.Test;
 
 import java.util.Arrays;
 import java.util.concurrent.TimeUnit;
 
 // Interactive auth with force_login and step-up MFA
 // https://identitydivision.visualstudio.com/DefaultCollection/IDDP/_workitems/edit/99656
+@Ignore("https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1886086")
```

Comment:
For this and any other AUTO_MFA test cases, Ryan was saying that we can try clicking on the "Call <Number>" button and see if that let's the flow proceed.

![image](https://user-images.githubusercontent.com/37125644/175165624-0e437d46-0226-4abe-afdc-af61c2bb2f69.png)

Can you try that out? If that works then we can remove the ignore annotation

Reply:
I tried this, didn't seem to work. the called number doesn't responds and the sign in request doesn't finish. Same thing with text, but that's expected since I need to enter the code myself.

Reply:
Probably need to follow up with Ryan on this